### PR TITLE
Add troubleshooting entry on failed webhook

### PR DESCRIPTION
**Pull request by sgibson91**
This adds the debugging steps taken in https://github.com/2i2c-org/infrastructure/pull/1448 to our troubleshooting guide

### CODE REVIEW

File: docs/topic/troubleshooting.md [hunk 1]
```diff
@@ -49,3 +52,28 @@ prometheus:
 The [default for this value is 2GB](https://github.com/2i2c-org/infrastructure/tree/HEAD/helm-charts/support/values.yaml#L38).
 Try increasing this value in small increments and deploying using `python deployer deploy-support {{CLUSTER_NAME}}` until the server successfully starts up.
 Then make a Pull Request with your updated config.
+
+(troubleshooting/validation-webhook)=
+## Failure calling validation webhook for `support` components
+
+After addressing [](troubleshooting/prometheus-oom), you may see the following error when redeploying the support chart:
```
Comment (sgibson91) [inline docs/topic/troubleshooting.md:59]: I'm actually not sure I've got referencing right here - I can never remember sphinx vs jupyter book vs something else syntax!

**Review by GeorgianaElena — APPROVED** [review]
🚀

File: docs/topic/troubleshooting.md [hunk 2]
```diff
@@ -49,3 +52,28 @@ prometheus:
...
+kubectl -A delete ValidatingWebhookConfiguration support-ingress-nginx-admission
+```
+
+where the `-A` flag will apply this command across all namespaces.
+
+````{tip}
+Find the webhooks running on a cluster by running:
+
+```bash
+kubectl get ValidatingWebhooksConfigurations
```
Comment (sgibson91) [inline docs/topic/troubleshooting.md:77]:
```suggestion
kubectl -A get ValidatingWebhooksConfigurations
```

[the remark this answers is not in the document]
Comment (sgibson91) [reply docs/topic/troubleshooting.md]: Done in [2424e2f](https://github.com/2i2c-org/infrastructure/pull/1450/commits/2424e2f88959c4297cea70655ed989fb9d231954)! I actually thought I'd already merged this so was getting ready to open a new PR 🤦🏻‍♀️ Will merge now though

**Pull request merged**